scripts/aave-v2/eth-mainnet/Deploy.s.sol

```diff
@@ -32,6 +33,8 @@ contract Deploy is Script, Config {
     uint256 constant DEFAULT_INITIAL_DEPOSIT = 1e8;
     uint256 constant WBTC_INITIAL_DEPOSIT = 1e6;
 
+    string constant VERSION = "1.1";
```

Comment:
can also use a uint, maybe a little bit simpler

Reply:
not for 1.1

Reply:
Does not follow semantic versioning though